modules/cco-ccoctl-creating-individually.adoc

```diff
@@ -32,91 +32,114 @@ Some `ccoctl` commands make AWS API calls to create or modify AWS resources. To
 $ ccoctl aws create-key-pair
 ----
 +
+[NOTE]
+====
+If your cluster uses Technology Preview features that are enabled by the `TechPreviewNoUpgrade` feature set, you must include the `--enable-tech-preview` parameter.
+====
++
```

Comment:
`ccoctl aws create-key-pair` does not use credentials request manifests. We can remove this section.